src/parser/mod.rs

```diff
@@ -5012,17 +5018,23 @@ impl<'a> Parser<'a> {
             None
         };
 
+        let click_house_config = if dialect_of!(self is ClickHouseDialect) {
```

Comment:
```suggestion
        let click_house_config = if dialect_of!(self is ClickHouseDialect | GenericDialect) {
```
if the behavior isn't conflicting with existing behavior on the generic dialect we can have the dialect accept it as well, we tend to do that when it allows https://github.com/sqlparser-rs/sqlparser-rs?tab=readme-ov-file#new-syntax

Reply:
Updated.